external/fv3fit/fv3fit/sklearn/_random_forest.py

```diff
@@ -85,15 +87,18 @@ def load(cls, path: str) -> "SklearnWrapper":
         return SklearnWrapper.load(path)
 
 
-class _RegressorEnsemble:
-    """Ensemble of regressors that are incrementally trained in batches
-
+class _RandomForestEnsemble:
+    """
+    Ensemble of RandomForestRegressor objects that are each trained on a separate
+    batch of data.
     """
 
     def __init__(
-        self, base_regressor, regressors: Sequence[sklearn.base.BaseEstimator] = None,
+        self,
+        hyperparameters: RandomForestHyperparameters,
+        regressors: Sequence[sklearn.base.BaseEstimator] = None,
     ) -> None:
-        self.base_regressor = base_regressor
+        self.hyperparameters = hyperparameters
```

Comment:
I did this refactor when I suspected this was the cause of non-reproducible runs, I can revert it if desired. However it does move in the direction we want to go with these classes (merging them, making them random forest-specific).